src/docs/pages/design-tokens.md

```diff
@@ -0,0 +1,44 @@
+---
+eleventyNavigation:
+  key: Design tokens
+  parent: Introduction
+  order: 9
+layout: layout.njk
+title: Design tokens
+permalink: introduction/design-tokens/
+---
+
+
+# Design tokens
+
+In order to keep design and implementation in sync Liquid uses so-called design tokens. These tokens are [hosted on Figma][liquid tokens on figma]. They include global variables such as colors, spacings etc. and end up as custom CSS properties in autogenerated CSS files which you eventually import in your own code.
+
+In some cases you might want to access the "raw" design tokens or even use your own design tokens in your project. Both is possible with Liquid. 
+
+## Accessing Liquid’s design tokens
+
+Maybe you don't want to use the Tailwind preset which is bundled with Liquid and want to build your own config from scratch. Or you are using something other than Tailwind for the same purpose. In such cases you might want to access Liquid's design tokens (colors, spacings etc.) as "raw data". You can do so by importing the bundled design token file under `@emdgroup-liquid/liquid/dist/css/design-tokens.json`.
+
+## Rolling your own design tokens
+
+You have at least two options:
+
+1. Copy and overwrite the `design-tokens.json` and / or the CSS files with your own manually.
+
+2. Copy [Liquid's token file on Figma][liquid tokens on figma], adjust it to your needs and auto-generate your `design-tokens.json` and a `liquid.globals.css` file with values from your own Figma token file by running a command from the `@emdgroup-liquid/liquid` package using [`npx`](https://docs.npmjs.com/cli/v7/commands/npx).
```

Comment:
Is `[Liquid's token file on Figma][liquid tokens on figma]` intended? I guess the link is probably missing here.

Reply:
The link is at the bottom of the file. This is intended.